src/app/zap-templates/zcl/data-model/chip/global-enums.xml

```diff
@@ -34,6 +34,18 @@ TODO: Make these structures global rather than defining them for each cluster.
     <item name="High" value="0x02"/>
     <item name="Automatic" value="0x03"/>
   </enum>
+
+  <enum name="StreamTypeEnum" type="enum8">
```

Comment:
This fixes https://github.com/project-chip/connectedhomeip/issues/35838

You'll need to update [webrtc-provider-cluster.xml](https://github.com/project-chip/connectedhomeip/blob/ca79d5910a50e6fb2f82748bd1540153d98601b6/src/app/zap-templates/zcl/data-model/chip/webrtc-provider-cluster.xml#L19) & remove from there

Reply:
That is also done with this PR.